README.md

```diff
@@ -21,27 +21,32 @@ To participate in this project, we ask you to abide by the OpenOakland [Code of
 
 # Quickstart for developers
 
-Run the woaq infrastructure with [Docker Compose](https://docs.docker.com/compose/)
-
-```bash
+Install [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/)  
+Run Locally  
+```
   git clone https://github.com/openoakland/woeip.git
   cd woeip
   docker-compose up -d --build
 ```
+Seed initial pollutant data as outlined in Issue #TBD 
 
-Browse the client at [lvh.me](http://lvh.me), ping the api at [api.lvh.me](http://api.lvh.me).
+## Services  
 
-## Services
-
-- woaq-proxy: [nginx-proxy](https://github.com/jwilder/nginx-proxy), [ssl-autogen](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion)
- 
+- woaq-proxy: [nginx-proxy](https://github.com/jwilder/nginx-proxy), [ssl-autogen](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion)   
 - woaq-web:
-[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [zurb](https://react.foundation/)
-
+[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [semantic UI](https://react.semantic-ui.com/)  
 - woaq-api: [python](https://www.python.org/), [django-rest-framework](https://www.django-rest-framework.org/)
-
 - woaq-db: [postgis](https://postgis.net/)
 
+## Domains 
+- Webpage: [lvh.me](http://lvh.me)
```

Comment:
How about `Homepage` instead of Webpage (more specific)? Yes, I'm being nitpicky.

Reply:
I see what you mean. And it is language I've been struggling to pin down. Really, I'm trying to highlight that it's frontend v. backend v. api documentation.

Reply:
In that case, it's okay to over-clarify, I think. For example:

> _Front-end:_
- Homepage (will eventually be woaq.org): lvh.me

> _Back-end:_
- Datavase: <link>
- API: <link>

Reply:
No right answer, here!